langner-ot-base-inventory/operations.py

```diff
@@ -0,0 +1,198 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+import requests, json
+from connectors.core.connector import ConnectorError, get_logger
+
+logger = get_logger('langner-ot-base-inventory')
+
+headers = {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json'
+}
+
+
+class LangnerOTBase(object):
+    def __init__(self, config, *args, **kwargs):
+        self.username = config.get('username')
+        self.password = config.get('password')
+        url = config.get('server_url').strip('/')
+        if not url.startswith('https://') and not url.startswith('http://'):
+            self.url = 'https://{0}/ot-base/api/v1/'.format(url)
+        else:
+            self.url = url + '/ot-base/api/v1/'
+        self.verify_ssl = config.get('verify_ssl')
+
+    def make_rest_call(self, url, method, data=None, params=None):
```

Comment:
The parameter name "url" of the method "make_rest_call" can have a better name. eg: endpoint